K2Bridge/KustoDAL/KustoDataAccess.cs

```diff
@@ -163,6 +174,66 @@ private void MapFieldCaps(IDataReader kustoResults, FieldCapabilityResponse resp
             }
         }
 
+        private async Task HandleDynamicField(FieldCapabilityResponse response, string tableName, FieldCapabilityElement fieldCapabilityElement)
```

Comment:
can you add some doc for this method. its not trivial so I guess it will help

Reply:
Added a comment, WDYT